README.md

```diff
@@ -75,6 +75,7 @@ Cassandra      | [Cassandra](http://cassandra.apache.org/) is a highly available
 EdgeFS         | [EdgeFS](http://edgefs.io) is high-performance and fault-tolerant object storage system with Geo-Transparent data access to file, block or object.                                                                                                                                                                     | edgefs.rook.io/v1beta1       | Beta
 Minio          | [Minio](https://www.minio.io/) is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.                                                                                                                                                                         | minio.rook.io/v1alpha1       | Alpha
 NFS            | [Network File System (NFS)](https://github.com/nfs-ganesha/nfs-ganesha/wiki) allows remote hosts to mount file systems over a network and interact with those file systems as though they are mounted locally.                                                                                                         | nfs.rook.io/v1alpha1         | Alpha
+NooBaa         | [NooBaa](https://github.com/noobaa/noobaa-core) is an object data service for hybrid and multi cloud environments. | noobaa.rook.io/v1alpha1 | Alpha
```

Comment:
I just followed previous rows, and the entire table renders fine since outer pipes are optional.
See [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables).
Resolving, thanks.